webapp/packages/core-connections/src/ConnectionExecutionContext/ConnectionExecutionContext.ts

```diff
@@ -8,10 +8,13 @@
 import { computed, makeObservable, observable } from 'mobx';
 
 import type { ITask, TaskScheduler } from '@cloudbeaver/core-executor';
+import type { AsyncTaskInfoService, GraphQLService } from '@cloudbeaver/core-sdk';
 
 import type { ConnectionExecutionContextResource, IConnectionExecutionContextInfo } from './ConnectionExecutionContextResource';
 import type { IConnectionExecutionContext } from './IConnectionExecutionContext';
 
+const DEFAULT_AUTO_COMMIT = true;
+
```

Comment:
This constant is defined by the backend and we actually don't know the default state. Can we request an API change so we will always have this value or can we hide commit actions if we can't get a current state?